src/wizards/modify_task.xml

```diff
@@ -156,6 +162,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             </xsl:if>
 
             <xsl:if test="(/wizard/params/param[name='reschedule']/value = 1) and (/wizard/params/param[name='start_year']/value != '') and (/wizard/params/param[name='start_month']/value != '') and (/wizard/params/param[name='start_day']/value != '') and (/wizard/params/param[name='start_hour']/value != '') and (/wizard/params/param[name='start_minute']/value != '')">
+              <xsl:variable name="start">
+                <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_year']/value), 2)"/>
```

Comment:
I think you could use the [`format-number()` function](https://www.w3schools.com/xml/func_formatnumber.asp) to format these numbers. Also, I'd like it better if the year was kept at 4 digits.

Reply:
Done in 7e96e38.